README.md

```diff
@@ -1,49 +1,61 @@
-## Development
+[![Build Status](https://travis-ci.org/dpordomingo/gitbase-playground.svg)](https://travis-ci.org/dpordomingo/gitbase-playground)
+[![codecov.io](https://codecov.io/github/dpordomingo/gitbase-playground/coverage.svg)](https://codecov.io/github/dpordomingo/gitbase-playground)
+![unstable](https://svg-badge.appspot.com/badge/stability/unstable?a)
 
-### Dependencies
+# Gitbase Playground
 
-Launch [bblfshd](https://github.com/bblfsh/bblfshd) and install the drivers. More info in the [bblfshd documentation](https://doc.bblf.sh/user/getting-started.html).
+Web application powered by [gitbase](https://github.com/src-d/gitbase), were you can query git repositories with a MySQL.
 
-```bash
-docker run -d --name bblfshd --privileged -p 9432:9432 -v /var/lib/bblfshd:/var/lib/bblfshd bblfsh/bblfshd
-docker exec -it bblfshd bblfshctl driver install --all
-```
+<!-- ![Screenshot](.github/screenshot.png?raw=true) //-->
 
-Install [gitbase](https://github.com/src-d/gitbase), populate a repository directory, and start it.
 
-```bash
-go get github.com/src-d/gitbase/...
-cd $GOPATH/src/github.com/src-d/gitbase
-make dependencies
-mkdir repos
-git clone https://github.com/src-d/gitbase-playground.git repos/gitbase-playground
-go run cli/gitbase/main.go server -v --git=repos
-```
+# Usage
 
-### Build
+## Dependencies
 
-```bash
-go build -o gitbase-playground cmd/server/main.go
-```
+The playground will run the queries against a [gitbase](https://github.com/src-d/gitbase) server, and will request UASTs to a [bblfsh](https://doc.bblf.sh/) server; both should be accessible for the playground; you can check its default [configuration values](docs/CONTRIBUTING.md#configuration).
 
-### Run
 
-Use `GITBASEPG_ENV=dev` for extra logs information.
+## Run the playground
 
-Development:
+You can run the app from a docker image, a released binary or installing and building the project.
 
-```bash
-GITBASEPG_ENV=dev go run cmd/server/main.go
-```
+Once the server is running &ndash;with its default values&ndash;, it will be accessible through: http://localhost:8080
 
-Built binary:
+Read [more about how to run bblfsh and gitbase dependencies](docs/quickstart.md).
```

Comment:
In my opinion we could include the contents here in the main readme. These are mandatory dependencies, without it you can't do anything. And the quickstart doc is not so big.

Reply:
I don't think the same.
Our [documentation guidelines](https://github.com/src-d/guide/blob/master/engineering/documentation.md#readmemd
) suggest to consider the `README.md` an "_overview of the project_", and suggest adding only "_basic installation instructions. If the instructions are long they should be part of a different document linked from this one._"
Our guides also consider the `README.md` as an "_index for all other documentation_" so it seems that using other docs for more details is something welcomed.

And I find another reason to do so: The instalation of the dependencies is not something "core" in the project, something so relevant that should be moved to the main `README.md`; on the other hand, if we move them to a separate doc, it can be considered as a minor thing more easily (edited, deleted...)

Reply:
Thanks for explaining the reasoning. I still find it confusing that the readme includes "incomplete" instructions. What is `GITBASE_CONTAINER` referring to a few lines below? As the guidelines say, maybe this should all be moved to one place only, the quickstart guide?

Anyway it's just a matter of opinion, up to you.

Reply:
Thanks for spotting the `GITBASE_CONTAINER` leftover (it came from the `quickstart.md`).
I fixed it to be consistent with the whole docker command: "_link with a `gitbase` container, and use it as a connection addres_".

Many thanks for taking your time in reviewing so deeply the` README.md` :+1:; I appreciate it a lot, and I'm sure that @campoy will do it too once this project gets more mature.